endpoint/endpoint.go

```diff
@@ -75,7 +75,17 @@ func (t Targets) Len() int {
 }
 
 func (t Targets) Less(i, j int) bool {
-	return t[i] < t[j]
+	ipi, err := netip.ParseAddr(t[i])
+	if err != nil {
+		return t[i] < t[j]
```

Comment:
this looks wrong, if `t[j]` is not parseable, or ?

Example which would be wrong:
```
t[j] := 2600.com 
t[i] := 1.1.1.1 // or use 3.3.3.3 if the dot1 is the one that is good
```